crates/cheatnet/src/lib.rs

```diff
@@ -12,28 +12,28 @@ pub mod panic_data;
 pub mod rpc;
 pub mod state;
 
-pub struct CheatnetState {
-    cheatcode_state: CheatcodeState,
-    pub blockifier_state: CachedState<ExtendedStateReader>,
-    pub deploy_salt_base: u32,
-}
+// pub struct CheatnetState {
```

Comment:
This file can be deleted then?

Reply:
Yup

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/753/commits/452e9c53c620532c3956679126781b6611fc052c